beacon-chain/cache/active_balance.go

```diff
@@ -0,0 +1,104 @@
+// +build !libfuzzer
+
+package cache
+
+import (
+	"encoding/binary"
+	"sync"
+
+	lru "github.com/hashicorp/golang-lru"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/prysmaticlabs/prysm/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+var (
+	// maxBalanceCacheSize defines the max number of active balance on per epoch basis can cache.
```

Comment:
This is the number of epochs worth of balances right? The wording is a bit tricky